### PR TITLE
47145_course-index

### PR DESCRIPTION
**Pull request by noor-framgia**

### CODE REVIEW

File: app/views/supervisor/courses/_courses.html.erb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+
+<tr>
+  <td><%= link_to course.name, [:supervisor, course] %></td>
+  <td><%= course.description.truncate 50 %></td>
+  <td><%= course.start_date %></td>
+  <td><%= course.end_date %></td>
+  <!--<td><%= course.users.supervisors.count %></td>-->
```
Comment (shakil-fg) [inline app/views/supervisor/courses/_courses.html.erb]: remove temporary comments

File: config/locales/en.yml [hunk 2]
```diff
@@ -72,3 +72,17 @@ en:
...
+  course_start_date:  "Start Date"
+  course_end_date:  "End Date"
+  course_status:  "Status"
+  course_detail: "Course Details"
+  assign_supervisor: "Assign Supervisor"
+  assign_trinee:  "Assign Trainee"
+
+
+
+
```
Comment (shakil-fg) [inline config/locales/en.yml]: remove all newlines at the end

File: config/locales/en.yml [hunk 3]
```diff
@@ -72,3 +72,17 @@ en:
...
+  all_course: "All Courses"
+  course_name: "Course Name"
+  course_description: "Course Description"
+  course_start_date:  "Start Date"
+  course_end_date:  "End Date"
+  course_status:  "Status"
+  course_detail: "Course Details"
+  assign_supervisor: "Assign Supervisor"
+  assign_trinee:  "Assign Trainee"
+
```
Comment (mdmahmudurrahman) [inline config/locales/en.yml]: Remove lines!

File: app/views/supervisor/courses/show.html.erb [hunk 4]
```diff
@@ -1,2 +1,46 @@
...
+      </thead>
+      <tbody>
+      <% @course.subjects.each do |subject| %>
+        <tr>
+          <td><%= link_to subject.name, [:supervisor, subject]  %></td>
+          <td><%= subject.description.truncate 50 %></td>
+        </tr>
+        </tbody>
+      <% end %>
+
```
Comment (mdmahmudurrahman) [inline app/views/supervisor/courses/show.html.erb]: Remove lines!

File: app/views/supervisor/courses/index.html.erb [hunk 5]
```diff
@@ -1 +1,29 @@
...
+    <tr>
+      <td><%= link_to course.name, [:supervisor, course]  %></td>
+      <td><%= course.description.truncate 50 %></td>
+      <td><%= course.start_date %></td>
+      <td><%= course.end_date %></td>
+      <td><%= course.is_open %></td>
+    </tr>
+  </tbody>
+  <% end %>
+
```
Comment (mdmahmudurrahman) [inline app/views/supervisor/courses/index.html.erb]: Remove lines!

File: app/views/supervisor/courses/show.html.erb [hunk 6]
```diff
@@ -1,2 +1,52 @@
-<h1>Supervisor::Courses#show</h1>
-<p>Find me in app/views/supervisor/courses/show.html.erb</p>
+<% provide(:title, @course.name) %>
```
Comment (dieunb) [inline app/views/supervisor/courses/show.html.erb]: remove `()`

**Pull request merged**